src/Actions/babylon.condition.ts

```diff
@@ -1,29 +1,67 @@
 ﻿module BABYLON {
+    /**
+     * A condition applied to an action 
+     */
     export class Condition {
+        /**
+         * Internal only - manager for action
+         * @ignore 
+         */
         public _actionManager: ActionManager;
 
+        /**
+         * Internal only
+         * @ignore 
+         */
         public _evaluationId: number;
+
+        /**
+         * Internal only
+         * @ignore 
+         */
         public _currentResult: boolean;
 
+        /**
+         * Creates a new {BABYLON.Condition}
+         * @param actionManager  
+         */
         constructor(actionManager: ActionManager) {
             this._actionManager = actionManager;
         }
 
+        /**
+         * Gets the current state of isValid

```

Comment:
Perhaps we could say something like Check if the current Condition is valid

Reply:
Done